### ISSUE TITLE
关于02-Logistic Regression/Logistic_Regression.py，这个模型应该只是普通的线性模型？

### ISSUE THREAD

**Issue by snjl** [SherlockLiao/pytorch-beginner#28]
您好，02-Logistic Regression/Logistic_Regression.py中34行直接将线性的结果输出了，那是否意味着这仅仅是一个线性模型，而非逻辑回归，逻辑回归应该必须有一个激活函数，类似relu或者sigmoid？

**Comment by L1aoXingyu** [comment]
logistice regression 就是 sigmoid(wx + b)，是一个二分类，而这里是一个10分类，所以sigmoid变成了softmax，softmax被吸收到 nn.CrossEntropyLoss() 里面了。

**Comment by snjl** [comment]
感谢您的解答！解决了一个困扰我的问题！
